src/lib_ccx/matroska.c

```diff
@@ -1099,15 +1099,24 @@ char *ass_ssa_sentence_erase_read_order(char *text)
 
 void save_sub_track(struct matroska_ctx *mkv_ctx, struct matroska_sub_track *track)
 {
-	char *filename = generate_filename_from_track(mkv_ctx, track);
-	mprint("\nOutput file: %s", filename);
```

Comment:
Don't think removing it in all cases even when not using stdout is a good idea

Reply:
I'm using an `if` statement to check if CCExtractor has been asked to output to standard output. Those two lines are present in the `else` block: [line #1110](https://github.com/CCExtractor/ccextractor/pull/1460/files/1f0eb852b269a01e5cc374ef26db060d42f9b7d3#diff-a404e3db34ed3c7c409c47344a2aa2668179b6004ce9857b6e625f54ec3d6c67R1111).